rootio/cmd/root-srv/main_go16.go

```diff
@@ -0,0 +1,77 @@
+// +build !go1.7
```

Comment:
same comment than for the other file :)
please move this at the correct location.